tests/unit/tests/4-list-table-split.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+set -x
+set -e
+
+pgcopydb list table-parts --source "${PGCOPYDB_SOURCE_PGURI}" \
```

Comment:
You can omit `--source` in that case, pgcopydb will just use the environment variable directly.